pipelines/e2e/README.md

```diff
@@ -18,6 +18,10 @@ affected by RHTAP services or which results could affect the RHTAP workflow.
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+### Changes in 1.9.0
+- modified the pipeline to dynamically source the `snapshot_spec.json`
```

Comment:
Capital M and `s/-/*/` to match the other entries please

Reply:
Fixed, thanks!